src/components/DatePicker/DatePicker.js

```diff
@@ -9,35 +9,82 @@ import Icon from '../Icon';
 import DatePickerWrapper from './styles/DatePickerWrapper';
 
 import DateRangePicker from './DateRangePicker';
+import moment from 'moment';
+moment.updateLocale('en', {
+  weekdaysMin: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
```

Comment:
Is this a global change? Can we contain it to a copy of moment used just for this component?

Reply:
It is, and it also only applies for English.

Reply:
It looks like we would have to hack around this limitation in a similarly ugly way. It might be better to leave this hacky code in here for now.